stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -286,10 +286,19 @@ public class MessageInputView : ConstraintLayout {
     }
 
     public fun setCommands(commands: List<Command>) {
-        suggestionListController?.commands = commands
+        suggestionListController?.commands = commands + requestDump()
```

Comment:
Perhaps we need to check if the `apiRequestsAnalyser` was registered?

Reply:
This got out dated